### PR TITLE
actually, do debug things

### PR DESCRIPTION
**Pull request by MrChico**

### CODE REVIEW

**Comment by MrChico** [comment]
I don't get why push fails and pull request succeeds

File: src/dapp-tests/integration/tests.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-set -e
+set -ex
```
Comment (asymmetric) [inline src/dapp-tests/integration/tests.sh:3]: Is this nice to have on CI outputs? Or locally?

Comment (MrChico) [reply src/dapp-tests/integration/tests.sh:3]: It's nice on ci outputs!

File: src/dapp-tests/integration/tests.sh [hunk 2]
```diff
@@ -23,7 +23,7 @@ dapp_testnet() {
 
   dapp testnet --dir "$TMPDIR" &
   # give it a few secs to start up
-  sleep 20
+  sleep 30
```
Comment (MrChico) [inline src/dapp-tests/integration/tests.sh:26]:
```suggestion
  sleep 60
```

**Comment by xwvvvvwx** [comment]
looks like this is just failing due to `solc-0.4.18` on macos, which should be fixed as of c351dde206dc8d6f3237118f251cffb18553f292

**Comment by MrChico** [comment]
Still failing for some mysterious reason

**Pull request merged**